modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -96,6 +96,15 @@ Egress firewall rules do not apply to traffic that goes through routers. Any use
 An egress firewall has the following limitations:
 
 * No project can have more than one {kind} object.
+ifdef::openshift-sdn[]
++
+[IMPORTANT]
+====
+The creation of more than one {kind} object is allowed, however it should not be done. When you create more than one {kind} object, the following message is returned: `dropping all rules`. In actuality, all external traffic is dropped, which can cause security risks for your organization.
+
+Users should not create more than one {kind} object.
```

Comment:
Is this sentence needed? It reads as a duplication of the above information to me (minus the justification)

Reply:
probably not. I was doing it just to reiterate it, but hopefully they get the memo.